ogticrd/covid-safe-paths
# Feat: getting advice data from service

#### Description:


Refactor advise.js file to receive all the recommendations from covid service instead of hard code in the app
#### Linked issues:


Fix: [254](https://trello.com/c/uJ3JD3h9/254-as-a-user-i-want-the-consejos-pdf-files-in-the-app-service-api-so-that-the-app-becomes-lighter-1)

#### How to test:


- `git pull && git checkout Feat/254-getting-advice-data-from-service`
- Head over to advice screen and you should see all the recommendations
